nautobot/extras/templates/extras/jobresult.html

```diff
@@ -80,8 +77,8 @@
     {% include 'extras/inc/jobresult.html' with result=result log_table=log_table %}
 {% endblock content_full_width_page %}
 
-{% block extra_tab_content %}
-    <div role="tabpanel" class="tab-pane" id="json">
+{% block advanced_content_full_width_page %}
```

Comment:
Seems like the contents of this block are already split into left and right columns, can we convert them to using `advanced_content_left_page` and `advanced_content_right_page` instead of `advanced_content_full_width_page` since they're not actually full-width content?

Reply:
There is no `advanced_content_left_page`. But I could add that.